itenium-socks/src/app/socks/sock.component.ts

```diff
@@ -3,22 +3,24 @@ import { Observable } from 'rxjs';
 import { Sock } from './sock.model';
 import { SocksService } from './socks.service';
 import { AsyncPipe, NgIf, TitleCasePipe } from '@angular/common';
+import { SockReviewsComponent } from "./sock-reviews.component";
 
 @Component({
   selector: 'app-sock',
   standalone: true,
-  imports: [NgIf, AsyncPipe, TitleCasePipe],
+  imports: [NgIf, AsyncPipe, TitleCasePipe, SockReviewsComponent],
   templateUrl: './sock.component.html'
 })
 export class SockComponent {
   sock$!: Observable<Sock>;
+  sockId: number | undefined;
 
   constructor(private socksService: SocksService) {}
 
   ngOnInit(): void {
     // HACK: This is not the way to get the sockId!!
-    const sockId = +window.location.pathname.split('/')[2];
-    this.sock$ = this.socksService.getById(sockId);
+    this.sockId = +window.location.pathname.split('/')[2];
```

Comment:
Probeer fields altijd te vermijden.

Een van de oefeningen is "Maak een next button om naar de volgende sock te gaan"
Hiervoor moet je dan een `ngOnChanges` gaan doen (en later nog andere dingen)

Je kan vb met een getter werken:
```ts
get sockId(): number {
  return +window.location.pathname.split('/')[2];
}
```

(maar natuurlijk is die window.location NOT the Angular way to go! (nog een andere oefn;)